### PR TITLE
Add copy outputs to outputs table

### PR DESCRIPTION
**Pull request by hunterwerlla**
## Screenshot
Kapture 2020-11-11 at 12 38 51

## related issues
#2179

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsTableView.kt [hunk 1]
```diff
@@ -9,15 +10,35 @@ import software.aws.toolkits.resources.message
...
-        DynamicTableView.Field(message("cloudformation.stack.outputs.value")) { it.outputValue() },
-        DynamicTableView.Field(message("cloudformation.stack.outputs.description")) { it.description() },
-        DynamicTableView.Field(message("cloudformation.stack.outputs.export")) { it.exportName() }
-    ).apply { component.border = JBUI.Borders.empty() }
+    private val key = DynamicTableView.Field<Output>(message("cloudformation.stack.outputs.key")) { it.outputKey() }
+    private val value = DynamicTableView.Field<Output>(message("cloudformation.stack.outputs.value")) { it.outputValue() }
+    private val description = DynamicTableView.Field<Output>(message("cloudformation.stack.outputs.description")) { it.description() }
+    private val export = DynamicTableView.Field<Output>(message("cloudformation.stack.outputs.export")) { it.exportName() }
+
+    private val table = DynamicTableView(
```
Comment (abrooksv) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsTableView.kt:18]: Fit on line line?

**Comment by abrooksv** [comment]
Seems a little funny they all have the copy icon IMO

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsTableView.kt (hunk 1 above)
Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsTableView.kt]: Sure

**Comment by hunterwerlla** [comment]
> Seems a little funny they all have the copy icon IMO

I mean maybe, but that's how it already was for the rest of the cfn table

**Review by abrooksv — APPROVED** [review]

**Comment by kiiadi** [comment]
I'm wondering if there's a way to change the default behaviour to copy-selected cell rather than copy row - I think that's what customers would expect 99% of the time.

**Comment by hunterwerlla** [comment]
> I'm wondering if there's a way to change the default behaviour to copy-selected cell rather than copy row - I think that's what customers would expect 99% of the time.

That is a good point, that is also easy to do, so changed

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsTableView.kt [hunk 2]
```diff
@@ -9,12 +10,12 @@ import software.aws.toolkits.resources.message
 import javax.swing.JComponent
 
 class OutputsTableView : View, OutputsListener, Disposable {
-    private val table = DynamicTableView<Output>(
-        DynamicTableView.Field(message("cloudformation.stack.outputs.key")) { it.outputKey() },
-        DynamicTableView.Field(message("cloudformation.stack.outputs.value")) { it.outputValue() },
-        DynamicTableView.Field(message("cloudformation.stack.outputs.description")) { it.description() },
-        DynamicTableView.Field(message("cloudformation.stack.outputs.export")) { it.exportName() }
-    ).apply { component.border = JBUI.Borders.empty() }
+    private val key = DynamicTableView.Field<Output>(message("cloudformation.stack.outputs.key")) { it.outputKey() }
```
Comment (kiiadi) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsTableView.kt:13]: Is this just for code-style? These no longer need to be broken out right?

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/ResourceTableView.kt [hunk 3]
```diff
@@ -25,17 +25,6 @@ class ResourceTableView : View, ResourceListener, Disposable {
         ) { it.resourceStatusReason() }
     ).apply { component.border = JBUI.Borders.empty() }
 
-    init {
-        table.addMouseListener(ResourceActionPopup(this::selected))
-    }
-
-    private fun selected(): SelectedResource? {
```
Comment (kiiadi) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/ResourceTableView.kt:32]: awesome - I hated myself for this

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/CloudFormationBrowserTest.kt [hunk 4]
```diff
@@ -129,19 +129,6 @@ class CloudFormationBrowserTest {
                     assertThat(createComplete).hasSize(1)
                     createComplete.first()
                 }
-
-                step("Check Logical ID action") {
-                    resource.click(MouseButton.RIGHT_BUTTON)
-                    findAndClick("//div[@text='Copy Logical ID']")
```
Comment (kiiadi) [inline ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/CloudFormationBrowserTest.kt:135]: the copy is actually in a few places I think ...

**Review by kiiadi — APPROVED** [review]

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsTableView.kt (hunk 2 above)
Comment (hunterwerlla) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/stack/OutputsTableView.kt:13]: Yeah this change can be reverted now, will do

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/CloudFormationBrowserTest.kt (hunk 4 above)
Comment (hunterwerlla) [reply ui-tests/tst/software/aws/toolkits/jetbrains/uitests/tests/CloudFormationBrowserTest.kt:135]: We have it in cloudwatch logs as well, but I'm not sure why I put it in there. Copying multiple rows/columns works as expected

**Pull request merged**